mk/win32/re.vcxproj

```diff
@@ -158,6 +158,7 @@
     <ClCompile Include="..\..\src\net\rt.c" />
     <ClCompile Include="..\..\src\net\sock.c" />
     <ClCompile Include="..\..\src\net\sockopt.c" />
+    <ClCompile Include="..\..\src\net\win32\hostaddr.c" />

```

Comment:
please also update src/net/mod.mk so that the code can be compiled with mingw32.